subfunctions/demo_CaBBI_FHN

```diff
@@ -0,0 +1,236 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% This script inverts the FHN model for the given fluorescence trace using
+% CaBBI method: [ Rahmati. et al. 2016, PLoS Comput Biol 12(2) ]
+%
+% To user:
+% I) in Step 1, determine the directory to the fluorescence trace (a row vector)
+% II) in Step 2, determine the sampling rate (Hz) used for recording the fluorescence trace
+% III) run the code
+% IV) the main outputs of the CaBBI are listed in Step 11. 
+%
+% Further options:
+% 1) In Step 10, if necessary, you can change the spike-detection threshold which is used to extract 
+%    spike-times from inferred voltage traces. Such change may increase the spike-detection accuracy. 
+% 2) In Step 6, if necessary, you can change some internal parameters of the optimization process.
+% 3) For further description of the toolbox options, please see:
+%    http://mbb-team.github.io/VBA-toolbox/wiki/Controlling-the-inversion-using-VBA-options/
+%    http://mbb-team.github.io/VBA-toolbox/wiki/VBA-output-structure/
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+clc; clear all; close all
+
+%% Step 1. Importing the fluorescence trace
+% import the fluorescence trace
+dataFolder = ['Fluor_Data/'];addpath('/Fluor_Data'); 
+Fluor_trace_name = 'fluorescence_data8';                                   % file name of the fluorescence trace
+Data_Directory = [dataFolder,Fluor_trace_name,'.mat'];                     
+ydata = importdata(Data_Directory);                                        % the fluorescence trace before pre-processing 
+
+
+%% Step 2. Determining the sampling rate used to record the fluorescence trace
+sampling_rate = 22.6;                                                        % in [Hz], for our data enter 22.6
+display(['the sampling rate used to record the fluorescence trace is << ',num2str(sampling_rate),' Hz >>']); %[Hz] 
+
+
+%% Step 3. Pre-processing
+% normalizing
+if size(ydata,1) ~= 1; ydata = ydata'; end                                 % fluorescence trace should be a row vector
+ydata = (ydata - mean(ydata))./(max(mean(ydata),1));                       % relativer fluorescncee trace
+ydata = ydata - median(ydata);                                             % removing the median
+
+% removing the slow drifts 
+xdata = 1:length(ydata);
+[xData, yData] = prepareCurveData( xdata, ydata );
```

Comment:
I added some lines to the code, so that it does not need the "prepareCurveData" function anymore.